src/PostgREST/Config.hs

```diff
@@ -273,6 +281,9 @@ readOptions = do
           |## stored proc that overrides the root "/" spec
           |## it must be inside the db-schema
           |# root-spec = "stored_proc_name"
+          |
+          |## content types to produce raw output
+          |#raw-output-media-type=["image/png","image/jpg"]
```

Comment:
Sorry, a nitpick. The config should be plural, end in `media-types` since it accepts a list.

Also, I think the name I proposed is too long. Ultimately we could shorten it to just `raw-media-types`.

Reply:
@steve-chavez makes sense. Will fix asap.